foodnet/membership/views.py

```diff
@@ -55,6 +55,31 @@ def invite(request):
     return render(request, 'foodnet/membership/invite.html', ctx)
 
 
+@login_required
+def departments_accounts(request, department_name=None):
+    # TODO: pagination of user profiles
+    assert department_name is not None
+    department = Department.objects.get(shortname=department_name)
+    user = UserProfile.get_for_user(request.user)
+    if not user.has_admin_permission(department=department):
+        return HttpResponseForbidden(content="Not a department admin.")
+
+    ctx = {'accounts': department.accounts}
+    return render(request, 'foodnet/department/accounts.html', ctx)
+
+
+@login_required
+def admin_profile(request, user_id=None):
+    # TODO: actually implement a profile form
+    assert user_id is not None
+    print("admin_profile edit for {}".format(user_id))
+    admin_user = UserProfile.get_for_user(request.user)
```

Comment:
The above views need tiding up.
Could you work on this branch a bit more please? Because to me it looks like this PR was created in a rush.